src/NumericalAlgorithms/Interpolation/Events/InterpolateWithoutInterpComponent.hpp

```diff
@@ -153,6 +169,27 @@ class InterpolateWithoutInterpComponent<VolumeDim, InterpolationTargetTag,
         temporal_id);
   }
 
+  template <typename ParallelComponent>
+  void operator()(
+      const typename InterpolationTargetTag::temporal_id::type& temporal_id,
+      const typename Tags::InterpPointInfo<Metavariables>::type& point_infos,
+      const Mesh<VolumeDim>& dg_mesh, const Mesh<VolumeDim>& subcell_mesh,
+      const evolution::dg::subcell::ActiveGrid active_grid,
```

Comment:
I think this library now must depend on `Evolution/DgSubcell` in the `CMakeLists.txt`. This probably will create a circular dependency that we will need to resolve at some point